helix-term/Cargo.toml

```diff
@@ -31,10 +31,11 @@ assets = [
 ]
 
 [features]
-default = ["git"]
+default = ["git", "steel"] # Remove steel if you don't want it
```

Comment:
For the initial merge let's disable this from `default` for now, and add instructions in `STEEL.md` on `cargo build` flag to enable